Avances en pantalla de inicio y productos con funcionalidad pero falt…

xavierGMFT opened the pull request:
Realize la Pantalla de Inicio 

Diseño con imagen de fondoa agregue los dos botones Servicios y Productos, que navegan a las pantallas correspondientes.
Pantallas de Productos y Servicios:
Implementé las pantallas para Productos y Servicios, conectadas a Firestore para mostrar datos dinámicos.
Validaciones para manejar estados de carga ("Cargando...") y ausencia de datos ("No hay datos disponibles").
Utilicé el componente reutilizable ListComponent para mostrar listas con imagen, nombre y precio.

xavierGMFT committed:
Avances en pantalla de inicio y productos con funcionalidad pero falta las tarjetas de detalle

arielGsDev reviewed (changes requested):
**Cambios requeridos**
![image](https://github.com/user-attachments/assets/6d4479e7-d32a-4b41-822a-989787c0cf6b)
este cambio en el enrutamiento no es necesario, la pantalla de services nisiquiera existe, la pantalla de products dejala mientras terminas ese modulo pq esa pantalla no deberia ir en el tabs
**ListComponent**
![image](https://github.com/user-attachments/assets/03ed20b1-e955-4140-89cc-0e1532d68237)
algunas propiedades de este prop, no son necesarias, usa la interfaz ProductI para tipar los props como un objeto y no tener que desesctructurar cada elemento 
![image](https://github.com/user-attachments/assets/dd217cfd-60ef-4c36-95f5-11aab42cdad1)
la parte de title no es necesaria, la parte de loading esta bien manejada, sin embargo cuando este los datos cargando usa un ActiveIndicator y como ya te dije los datos deben estar tipados como un ProductI[ ], para luego usar las propiedades como product.image o product.name 
![image](https://github.com/user-attachments/assets/2297a635-7c22-4e6a-b236-8a35f9cd8302)
El nombre del componente debe ser mas descriptivo, algo como ProductListComponent y lo de React.FC quitalo, eso esta practicamente en desuso 
![image](https://github.com/user-attachments/assets/a586189e-96f4-43df-82e3-7e3d8b6990f6)
en este caso el title no deberia ser un prop sino una variable dentro del componente y luego lo indexas ahi, para iterar en los resultados usa ScrollView en vez de FlatList
**Collections**
![image](https://github.com/user-attachments/assets/09da4e44-cdc7-47cf-ab46-beaae41857dc)
la adicion de SERVICES no es necesaria
**CategoryI**
![image](https://github.com/user-attachments/assets/1ea309ed-8eed-4ece-a2bf-25f78bd6f1e7)
No es necesaria una image 
**HomeScreen**
![image](https://github.com/user-attachments/assets/51218d07-795c-4776-baa5-383a18f9b587)
Reemplaza los botones de productos y de servicios por estos
Finalmente la pantalla de Products y Services no son necesarias

xavierGMFT committed:
Correcion del push request, como ya no era nesesario las pantalla de servicios y productos las elimine tambien el component list porque ya no era necesario ya que era para las otras pantallas, cambie el diseño de los botones por los de la app de ejemplo

arielGsDev committed:
falta imagen de fondo

arielGsDev reviewed (changes requested):
Añadir una imagen de fondo y ya

xavierGMFT committed:
Please enter a commit message to explain why this merge is necessary,
Merge branch 'develop' into develop-alejandro
Para bajarme los cambios hechos



OB

xavierGMFT committed:
Agregado la imagen de fondo

xavierGMFT committed:
Merge branch 'develop-alejandro' of https://github.com/ArielGavilanes/app-react-native-vinculacion into develop-alejandro

arielGsDev committed:
interface arreglo

arielGsDev committed:
Merge branch 'develop' of https://github.com/ArielGavilanes/app-react-native-vinculacion into develop-alejandro

arielGsDev committed:
Merge branch 'develop' of https://github.com/ArielGavilanes/app-react-native-vinculacion into develop-alejandro
